Contributing.md

```diff
@@ -0,0 +1,92 @@
+# Contribution Guide
+
+If you're reading this, you're probably creating a Pull Request or planning to do so and that's great!🥳
+
+# How to Contribute
+
+1. Fork this repository.
+
+2. Clone the forked repository.
+
+   ```bash
+   git clone 'https://github.com/bsoc-bitbyte/busify.git'
+   ```
+
+3. Navigate to the project directory.
+
+   ```bash
+   cd busify
+   ```
```

Comment:
After this step, there are a few more steps for the local setup

- Run the Postgres docker container with the command (make sure you have docker installed before running this)

  ```bash
   docker compose up
   ```
- In new terminal type

    ```bash
     cd backend
     npm install
     npm run start:dev
   ```
     This will start the backend server.

- Open another new terminal and type

    ```bash
     cd frontend
     npm install
     npm run dev
   ```
     This will start the frontend server.